src/Command/PurgeCommand.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+namespace Terminal42\LeadsBundle\Command;
+
+use Contao\CoreBundle\Framework\ContaoFrameworkInterface;
+use Contao\CoreBundle\Monolog\ContaoContext;
+use Contao\File;
+use Contao\FilesModel;
+use Contao\System;
+use Contao\Validator;
+use Doctrine\DBAL\Connection;
+use Psr\Log\LogLevel;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Filesystem\Filesystem;
+use Terminal42\LeadsBundle\Leads;
+use \Exception;
+
+class PurgeCommand extends Command
+{
+    /**
+     * @var ContaoFrameworkInterface
+     */
+    private $framework;
+
+    /**
+     * @var Connection
+     */
+    private $db;
+
+    /**
+     * @var Filesystem
+     */
+    private $fs;
+
+    /**
+     * Constructor.
+     *
+     * @param ContaoFrameworkInterface $framework
+     * @param Connection $db
+     * @param Filesystem $fs
+     */
+    public function __construct(ContaoFrameworkInterface $framework, Connection $db, Filesystem $fs = null)
+    {
+        if (null === $fs) {
+            $fs = new Filesystem();
+        }
+
+        $this->framework = $framework;
+        $this->db = $db;
+        $this->fs = $fs;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('leads:purge')
+            ->setDescription('Purge the leads outsite the configured storage period.');
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int|null|void
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->framework->initialize();
+
+        $purged = $this->executeBatchPurge();
+
+        if ($purged) {
+            $logMessage = 'The leads have been purged successfully.';
```

Comment:
Can we log the number of purged leads? Then we might even unify the message.

Reply:
the number of purged leads are logged for every master form - this message is just the final log message